js/preserveData.js

```diff
@@ -0,0 +1,21 @@
+const userName = document.querySelector('#userName');
+const userEmail = document.querySelector('#email');
+const userMessage = document.querySelector('#message');
+const submissionBtn = document.querySelector('#submit');
+
+window.addEventListener('load', () => {
+  const getUserData = JSON.parse(localStorage.getItem('setUserData'));
+  userName.value = getUserData.name;
+  userEmail.value = getUserData.email;
+  userMessage.value = getUserData.msg;
+});
+
+// set userDate to localStorage
+const setUserData = { name: '', email: '', msg: '' };
+
+submissionBtn.addEventListener('click', () => {
+  setUserData.name = userName.value;
+  setUserData.email = userEmail.value;
+  setUserData.msg = userMessage.value;
+  localStorage.setItem('setUserData', JSON.stringify(setUserData));
+});
```

Comment:
- [ ] Good job @ghermaico135 😃 , as the [previous reviewer](https://github.com/ghermaico135/Portfolio/pull/13#:~:text=However%20data%20should%20be%20saved%20in%20the%20browser%20when%20the%20user%20changes%20the%20content%20of%20any%20input%20field.%20But%20you%27re%20saving%20the%20data%20when%20the%20user%20clicks%20on%20the%20button%20submit%2C%20Kindly%20check%20it) said,  data should be saved in the browser when the user changes the content of any input field. But you're saving the data when the user clicks on the button submit, which means that the data will be saved when the user will be changing the content of any input field, not only when he clicks on the button submit. Kindly check it 👍